cmd/dra-example-kubeletplugin/main.go

```diff
@@ -141,6 +142,8 @@ func newApp() *cli.App {
 }
 
 func StartPlugin(ctx context.Context, config *Config) error {
+	log := klog.FromContext(ctx)
```

Comment:
It's odd to have a function called "Start" which blocks. Can we renamed to "Run"?

The functions also, with and without this change, ignores cancellation of the context that it is passed. If the context is only for passing a logger, then let's pass a logger instead of the context. Or honor cancellation...